src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -62,42 +62,37 @@ export default {
         width: '100%',
         maxWidth: '100%',
       }),
-    }
 
-    if (disabled) {
-      return {
-        ...rules,
-        ...disabledStyle,
-      }
-    }
-
-    return {
-      ...rules,
+      ...(disabled
```

Comment:
Nit: may be more maintainable to break this out into another function rather than inlining it.